ui/src/app/components/add-new-column/add-new-column.component.html

```diff
@@ -0,0 +1,39 @@
+<div mat-dialog-content>
+    <form [formGroup]="addNewColumnForm" class="add-new-column-form">
+        <h2>Column Details</h2>
+        <mat-form-field class="full-width" appearance="outline">
+            <mat-label>Name</mat-label>
+            <input matInput placeholder="Column Name" type="text" formControlName="name"/>
+        </mat-form-field>
+        <br>
+        <mat-form-field appearance="outline">
+            <mat-label>Datatype</mat-label>
+            <mat-select [(ngModel)]="selectedDatatype" class="input-field" formControlName="datatype" required="true" (selectionChange)="changeValidator()">
+                <mat-option *ngFor="let datatype of datatypes" [value]="datatype">{{datatype}}</mat-option>
+            </mat-select>
+        </mat-form-field>
+        <br>
+        <div *ngIf="selectedDatatype ==='STRING' || selectedDatatype ==='VARCHAR' || selectedDatatype ==='BYTES'">
```

Comment:
Nit: Can we move this logic out to `.ts` file as a method? Long conditions inside the HTML makes reduces readability.